allure-pytest/test/steps/outside/run_step_outside_test.py

```diff
@@ -0,0 +1,17 @@
+"""
+>>> getfixture('inject_matchers')
+>>> allure_report = getfixture('allure_report')
+"""
```

Comment:
this part is not necessary. Matcher injects as auto-used fixture, report fixture requested in test

Reply:
Ok! But actually this is like this on  this test:

https://github.com/allure-framework/allure-python/blob/ca105cb5ffc18814b8ef832e3d808ee5bb004563/allure-pytest/test/steps/outside/outside_steps_test.py#L1

Should it be removed there too?